src/details/ArborX_DetailsCartesianGrid.hpp

```diff
@@ -118,6 +121,23 @@ struct CartesianGrid
       m /= _n[d - 1];
       ARBORX_ASSERT(_n[d] < m);
     }
+
+    // Catch a potential loss of precision that may happen in cellBox() and can
+    // lead to wrong results.
+    //
+    // The machine precision by itself is not sufficient. In some experiments
+    // run with a full NGSIM datasets, values below 3 could still produce wrong
+    // results. This may still not be conservative enough, but all runs passed
+    // verification when this warning was not triggered.
+    float constexpr eps = 5 * std::numeric_limits<float>::epsilon();
```

Comment:
```suggestion
    constexpr auto eps = 5 * std::numeric_limits<float>::epsilon();
```
We use "constexpr West" and here `float` is redundant